app/templates/cookies/cookie_settings.html

```diff
@@ -0,0 +1,123 @@
+      </p>
+
+      <h2 class="govuk-heading-l">Cookie settings</h2>
+      <div class="cookie-settings__no-js">
+        <div class="govuk-govspeak">
+          <p>We use Javascript to set most of our cookies. Unfortunately Javascript is not running on your browser, so you cannot change your settings. You can try:</p>
+          <ul>
+            <li>reloading the page</li>
+            <li>turning on Javascript in your browser</li>
+          </ul>
+        </div>
+      </div>
+
+      <div class="cookie-settings__form-wrapper">
+        <p class="govuk-body">We use two types of cookies.</p>
+
+        <form id="cookie-settings">
+          <h3 class="govuk-heading-m">Cookies that measure website use</h3>
+
+          <p class="govuk-body">
+            We use Google Analytics to measure how you use the Digital Marketplace so we can improve it based on user needs.
+            Google Analytics sets cookies that store anonymised information about:
+          </p>
+
+          <ul class="govuk-list govuk-list--bullet">
+            <li>how you got to Digital Marketplace</li>
+            <li>the pages you visit on Digital Marketplace and government digital services, and how long you spend on each page</li>
+            <li>what you click on while you&rsquo;re using the service</li>
+          </ul>
+
+          <p class="govuk-body">
+            We do not allow Google to use or share the data about how you use this site.
+          </p>
+
+          <p class="govuk-body">
+            You can choose whether you&rsquo;re happy for us to use these cookies.
+          </p>
+          {% with
+              type = "radio",
+              name = "cookies-usage",
+              inline = true,
+              options = [
+                {
+                    "label": "On"
+                },
+                {
+                    "label": "Off"
+                }
+            ]
+          %}
+            {% include "toolkit/forms/selection-buttons.html" %}
+          {% endwith %}
+
+          <h3 class="govuk-heading-m">Strictly necessary cookies</h3>
+          <p class="govuk-body">These essential cookies do things like:</p>
+          <ul class="govuk-list govuk-list--bullet">
+            <li>storing sessions on your computer to help keep your information secure while you use the service</li>
+            <li>testing whether your browser is able to accept session cookies, allowing us to show a useful error message if you try to log in without cookies enabled</li>
+            <li>remembering the notifications you&rsquo;ve seen so we do not show them to you again</li>
+          </ul>
+          <p class="govuk-body"><a href="{{ url_for('external.cookies') }}">Find out more about cookies on Digital Marketplace</a></p>
+          {{ govukButton({
+            "text": "Save changes",
+          }) }}
+        </form>
+      </div>
+  </div>
+</div>
+{% endblock %}
```

Comment:
Can we move the strictly necessary cookies section to above the optional ones please?